src/HttpServer.cc

```diff
@@ -586,7 +586,7 @@ void Http2Handler::start_settings_timer() {
 
 int Http2Handler::fill_wb() {
   if (data_pending_) {
-    auto n = std::min(wb_.wleft(), data_pendinglen_);
+    auto n = (std::min)(wb_.wleft(), data_pendinglen_);
```

Comment:
No more stuffs, I just find two ways in google, one is add the extra parentheses, or defined `#define NOMINMAX `before including windows.h.